### PR TITLE
OSDOCS-5826: MicroShift networking release note

### PR DESCRIPTION
**Pull request by kelbrown20**
**For Version:** 4.13 
**Issue:** [OSDOCS-5826](https://issues.redhat.com//browse/OSDOCS-5825)

**Description:** Update release notes to show new Networking support docs

**Preview:** 

- [MicroShift release notes -> Networking ](https://59273--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-13-release-notes.html#microshift-4-13-networking)

**QE review:**

- [x] QE approval

### CODE REVIEW

**Comment by kelbrown20** [comment]
@jogeo Hey John! No rush for this, can you take a look at these Networking release notes when you get the chance?

File: microshift_release_notes/microshift-4-13-release-notes.adoc [hunk 1]
```diff
@@ -80,8 +80,18 @@ For more information, see xref:../microshift_support/microshift-etcd.html#micros
...
+
+[id="microshift-4-13-load-balancer"]
+==== Deploying network load balancers on {product-title}
+
+{product-title} now offers a built-in implementation of network load balancers. For more details, see xref:../microshift_networking/microshift-networking.adoc#microshift-deploying-a-load-balancer_microshift-networking[Deploying a TCP load balancer on a workload].
+
+[id="microshift-4-13-blocking-nodeport"]
+==== Blocking a NodePort service on a host interface 
+
+You can now block a NodePort service on a specific host interface and restrict access from outside a {product-title} node. For more information, see  xref:../microshift_networking/microshift-networking.html#microshift-blocking-nodeport-access_microshift-networking[Blocking external access to the NodePort service on a specific host interface].
```
Comment (opayne1) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:94]: Should NodePort be in backticks? I have seen it written both ways in the docs.

**Review by opayne1 — COMMENTED** [review]
Everything looks great! Just one question! Also, I am not seeing the changes I merged to storage. Probably just needs a rebase. I don't _think_ it would mess with anything at merging but I am not 100%.

File: microshift_release_notes/microshift-4-13-release-notes.adoc (hunk 1 above)
Comment (kelbrown20) [reply microshift_release_notes/microshift-4-13-release-notes.adoc:94]: Yeah this part was a bit confusing when writing the main docs. In my understanding, its in backtics when referenincg the NodePort as a variable  But when writing it as "NodePort service" in sentence format, its without. Thats what is like in the Kubernetes docs as well.

**Comment by kelbrown20** [comment]
@dhellmann @jogeo Hello! Can you guys take a look at this release note for MicroShift when you get the chance? Thank you!

File: microshift_release_notes/microshift-4-13-release-notes.adoc (hunk 1 above)
Comment (dhellmann) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:94]: This isn't really a new feature. The procedure describes how to use existing tools outside of MicroShift (nft) to accomplish the goal.

Comment (kelbrown20) [reply microshift_release_notes/microshift-4-13-release-notes.adoc:94]: Yeah thats a great point, its not really needed. I'll remove this note from the section. Thank you!

**Review by dhellmann — COMMENTED** [review]
/lgtm

**Comment by anuragthehatter** [comment]
@kelbrown20 Unless I missed it somewhere, I guess its imperative to mention that Ushift uses OVN-Kubernetes Local gateway mode. probaly somwhere under networking section @dhellmann Correct me in case

**Comment by dhellmann** [comment]
> @kelbrown20 Unless I missed it somewhere, I guess its imperative to mention that Ushift uses OVN-Kubernetes Local gateway mode. probaly somwhere under networking section @dhellmann Correct me in case

Can you give more background about why you think that is important? We do have a network architecture diagram in the documentation, but I'm not sure why we would need to call out that one detail.

**Comment by anuragthehatter** [comment]
@dhellmann This can be just a one liner note for the readers as mentioning just local gateway mode can explain lot of things wrt what ushift is capable of and what it might be limited to. Just a thought, specially for readers who hav experience with OCP dealing with both the modes. I am ok if you think arch diag itself is sufficient. Thanks

**Comment by jogeo** [comment]
@kelbrown20 I'll hand off the lgtm from QE to @anuragthehatter.

**Comment by anuragthehatter** [comment]
Thanks @dhellmann Sounds good to me

/lgtm

**Pull request merged**